src/carousel.js

```diff
@@ -866,6 +868,7 @@ const Carousel = createReactClass({
         top: props.vertical ? this.getTargetLeft() : 0,
       },
       function() {
+        self.props.onResize();
```

Comment:
Do we need to call `onResize` on every `setDimensions` call?

I was imagining a more honed refactoring of:

```js
setDimensions(stateCb) {
  // stuff
  this.setState(STUFF, stateCb || function () {});
}

onResize() {
  // stuff
  this.setDimensions(this.props.onResize);
}
```

Reply:
Yes you're right I didn't realize this was being called from other places. Updating